app/services/data_exports/data_storage.rb

```diff
@@ -0,0 +1,125 @@
+require 'fileutils'
+require 'securerandom'
+
+module DataExports
+  class NoStoredFilesFoundError < StandardError; end
+
+  class DataStorage
+    # Public: downloads all transcription files for a given transcription
+    # returns path to zip file
+    def zip_transcription_files(transcription)
+      Dir.mktmpdir { |directory_path|
+        if transcription.files.attached?
```

Comment:
this guard clause should be the first line of the method before you setup a tmpdir

Reply:
Ah, very true.